qiita_db/environment_manager.py

```diff
@@ -231,11 +231,18 @@ def make_environment(load_ontologies, download_reference, add_demo_user):
             ('demo@microbio.me', 4,
              '$2a$12$gnUi8Qg.0tvW243v889BhOBhWLIHyIJjjgaG6dxuRJkUM8nXG9Efe',
              'Demo', 'Qitta Dev', '1345 Colorado Avenue', '303-492-1984')""")
-        conn.execute("""
+        analysis_id = conn.execute_fetchone("""
             INSERT INTO qiita.analysis (email, name, description, dflt,
                                         analysis_status_id)
             VALUES
-            ('demo@microbio.me', 'demo@microbio.me-dflt', 'dflt', 't', 1)""")
+            ('demo@microbio.me', 'demo@microbio.me-dflt', 'dflt', 't', 1)
+            RETURNING analysis_id
+        """)[0]
+        portal_id = convert_to_id(qiita_config.portal, 'portal_type', 'portal')
```

Comment:
What if `qiita_config.portal` is not `QIITA`? This code is incomplete. The default analysis has to be added to all the available portals...

@squirrelo please pay attention at the code that you're writing...